### PR TITLE
add French translation

### PR DESCRIPTION
**Pull request by NedlawA**

### CODE REVIEW

**Comment by nminchow** [comment]
Hey, sorry for the delay here. Will test this tomorrow!

File: src/i18n/type-transformer.ts [hunk 1]
```diff
@@ -36,6 +37,10 @@ export const localeMapping: LocaleMapping = {
   [Locale.PortugueseBR]: {
     staticMapping: br,
     locale: 'br',
+  },
+  [Locale.FrenchFR]: {
```
Comment (nminchow) [inline src/i18n/type-transformer.ts]: This looks like its just `Locale.French` in the typedef.

Comment (nminchow) [reply src/i18n/type-transformer.ts]: Be sure to add this to the "supportedLocale" list!

**Review by nminchow — COMMENTED** [review]
Thanks for this PR! Before merge, we'll also need to run `yarn run i18n` to generate the automatic type files!

**Comment by nminchow** [comment]
If you'd rather I make these changes, I'm happy to merge this PR and do the cleanup myself. Let me know!

**Comment by NedlawA** [comment]
I think I got it right now. Let me know if it's still missing anything, I'll be happy to clean it up. Thanks for putting this together!

**Comment by nminchow** [comment]
Looks great! Thanks again! Let me know your discord username if you'd like the contributor role!

**Review by nminchow — APPROVED** [review]

**Pull request merged**